src/index.ts

```diff
@@ -72,6 +73,43 @@ export = (app: Probot) => {
     };
     await k8sApi.createNamespacedSecret(namespace,secret);
   });
+
+  //handle push event
+  app.on("push", async (context) => {
+
+    const configExists = Boolean(context.payload.commits?.reduce((acc, commit) => [...acc, ...commit.added, ...commit.modified,...commit.removed], []).find((name: String) => name == "peribolos.yaml"));
+    if(!configExists) {
+      return;
+    }
+    const payload = {
+      apiVersion: "tekton.dev/v1beta1",
+      kind: "TaskRun",
+      metadata: {
+        generateName: "peribolos-push-task-"
+      },
+      spec: {
+        taskRef:{
+          name: "peribolos-run"
+        },
+        params: [
+          { name: "REPO_NAME", value: ".github"},
+          { name: "INSTALLATION_ID", value: context.payload.installation?.id.toString()}
+        ],
+
+      }
+    }
+    try {
+      await client.createNamespacedCustomObject(
+        "tekton.dev",
+        "v1beta1",
+        namespace,
+        "taskruns",
+        payload
+      )
+    }catch(e: any){
+      app.log.error(e.message);
```

Comment:
asking from a perspective of nodejs rookie :sweat_smile: 
any specific reason of using try block here , instead of the `then` method of function?